webservices/utils.py

```diff
@@ -442,8 +448,59 @@ def get_current_cycle():
     return year + year % 2
 
 
+ES_SERVICE_INSTANCE_NAME = "fec-api-elasticsearch"
+AWS_ES_SERVICE = "es"
+REGION = "us-gov-west-1"
+PORT = 443
+
+
+def get_service_instance(service_instance_name):
+    return env.get_service(name=service_instance_name)
+
+
+def get_service_instance_credentials(service_instance):
+    return service_instance.credentials
+
+
+def create_es_client():
+    try:
+        es_service = get_service_instance(ES_SERVICE_INSTANCE_NAME)
+        if es_service:
+            credentials = es_service.credentials
+            #  create "http_auth".
+            host = credentials["host"]
+            access_key = credentials["access_key"]
+            secret_key = credentials["secret_key"]
+            aws_auth = AWS4Auth(access_key, secret_key, REGION, AWS_ES_SERVICE)
+
+            #  create elasticsearch client through "http_auth"
+            es_client = Elasticsearch(
+                hosts=[{"host": host, "port": PORT}],
+                http_auth=aws_auth,
+                use_ssl=True,
+                verify_certs=True,
+                ca_certs=certifi.where(),
+                connection_class=RequestsHttpConnection,
+                timeout=40,
+                max_retries=10,
+                retry_on_timeout=True,
+            )
+        else:
+            # create local elasticsearch client
+            url = 'http://localhost:9200'
+            es_client = Elasticsearch(
+                url,
+                timeout=30,
+                max_retries=10,
+                retry_on_timeout=True,
+            )
+        return es_client
+    except Exception as err:
+        logger.error('An error occurred trying to create Elasticsearch client.{0}'.format(err))
+
+
 def get_elasticsearch_connection():
-    es_conn = env.get_service(name='fec-api-search56')
+    es_conn = env.get_service(name='fec-api-elasticsearch')
```

Comment:
Should this use `ES_SERVICE_INSTANCE_NAME`?

Reply:
This is obsolete code forgot to cleanup. Thanks for flagging.